FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Jobs/Models/FhirToDataLakeOrchestratorJobResult.cs

```diff
@@ -30,6 +30,16 @@ public class FhirToDataLakeOrchestratorJobResult
         /// </summary>
         public long CompletedJobCount { get; set; }
 
+        /// <summary>
+        /// Processing jobs in the process of submitting, used for system scope
+        /// </summary>
+        public string[] SubmittingProcessingJob { get; set; } = null;
```

Comment:
how about adding comments for the fields only used for v4, and not used for v4 now?